### ISSUE TITLE
Add Overview Section for Spring Boot Starter on Pi4J Website

### ISSUE THREAD

**Issue by dariuszzbyrad** [Pi4J/pi4j.github.io#75]
Enhance the Pi4J website by adding a detailed overview section for the Pi4J Spring Boot Starter. The overview should highlight its purpose, core features, prerequisites, and usage, including how it simplifies integrating Pi4J into Spring Boot applications. It should also provide a clear description of the multi-module structure and links to resources like sample applications and GitHub repositories. This will ensure better visibility and understanding for developers visiting the website.

**Comment by FDelporte** [comment]
Good idea! Although we haven't released it yet, we could already start documentation to drive up "the fuzz." I should contact @dashaun again to push this forward and have it completed as a ready-to-use Spring Boot Starter...
